### ISSUE TITLE
403 access  denied

### ISSUE THREAD

**Issue by ShaolinGardener** [DiligentGuyHub/crowdfunding-platform#1]
This is not really an issue with the code, per se, but with my deployment. I've published to folder and then move the files up to the server. I runs just fine locally from VS but on the server I'm getting a 403. Any ideas? The code is awesome, btw. Thanks for posting it.

**Comment by DiligentGuyHub** [comment]
It may depend on the server, that you are using for application deployment. I issued a kind of a similar problem (i don't remember in details), but after I moved the solution to Microsoft Azure, the issue was solved somehow.
(But please be careful with Azure prices, I used 1 month free trial)
Thanks for kind words and good luck!

**Comment by DiligentGuyHub** [comment]
Also there are some issues with usage of the ASP.NET Identity Authorization (I used it for both registration by own credentials and for authorizing via google). I would prefer to refactor auth a lot...

**Comment by ShaolinGardener** [comment]
Yeah this is the issue I've reached now. Figured out the first one, was
deploying wrong type

Thanks for getting back to me.

Chaz

**Comment by DiligentGuyHub** [comment]
I would personally recommend you to follow this guy https://www.youtube.com/c/SingletonSean
He makes very high quality videos related to C#/.NET (even though he uses WPF, it still gives a good level of understanding applications architecture)

Viktor

**Comment by ShaolinGardener** [comment]
Cool thanks
